### PR TITLE
CLDR fixes for Denmark (danish)

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | We do never use kr. on websites. Because kr. is "used" in Sweden, Norway, Irland and Denmark. If a customer from Norway visits the website, the customer will think that the price is in his currency. Also I did some fixes for other related CLDR for Denmark. I am danish, therefor I am sure on this.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | yes, will add later
| How to test?  | kr. should be DKK

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9413)

### CODE REVIEW

**Comment by MathiasReker** [comment]
I don't understand why the test failed?

**Comment by PierreRambaud** [comment]
@Mreker It's not your fault, we have some troubles with Travis, too many PR, to many commits, it can't follow us :D

File: localization/dk.xml [hunk 1]
```diff
@@ -64,64 +64,6 @@
...
-      <taxRule iso_code_country="at" id_tax="1"/>
-      <taxRule iso_code_country="pl" id_tax="1"/>
-      <taxRule iso_code_country="pt" id_tax="1"/>
-      <taxRule iso_code_country="ro" id_tax="1"/>
-      <taxRule iso_code_country="si" id_tax="1"/>
-      <taxRule iso_code_country="sk" id_tax="1"/>
-      <taxRule iso_code_country="fi" id_tax="1"/>
-      <taxRule iso_code_country="se" id_tax="1"/>
-      <taxRule iso_code_country="uk" id_tax="1"/>
-    </taxRulesGroup>
```
Comment (eternoendless) [inline localization/dk.xml:124]: Why did you remove these taxes?

**Comment by eternoendless** [comment]
First of all, thank you for your contribution.

I have confirmed with a danish member of our team that prices in Denmark are labelled using "DSK" instead of "kr.". But there are several issues here.

1. Files in the `localization/CLDR` folder aren't used yet as they belong to the new implementation of CLDR that is not finished (but I hope it will for 1.7.5). So modifying them won't have any impact.
2. Even if those files _were_ used, they **must not be modified** as they come straight from the [Unicode CLDR project](http://cldr.unicode.org/index). Any change made there will be lost when we upgrade to the next version (we are already 2 versions behind, but the newer versions still contain this "mistake").
  The proper place to suggest this changes would be the Unicode consortium itself.
3. Prices in PrestaShop are currently handled using an old version of CLDR which is json-encoded and zipped in `translations/cldr/core.zip`. However, the rule of "CLDR files should not be modified" also applies here.
4. This also leads me to think that you may have not tested this contribution properly before submitting it, since your changes (predictably) don't seem to have any impact on how the prices are shown on the front office:
  screen shot 2018-08-09 at 12 20 37

The good news is, in the near future (maybe 1.7.6?) you will be able to add and customize how currencies are displayed in your shop, directly from the Back Office.

Please bear with us while we get there :)

File: localization/dk.xml (hunk 1 above)
Comment (MathiasReker) [reply localization/dk.xml:124]: We do only have one tax in Denmark. As you can see the taxes are all 25%.

**Comment by MathiasReker** [comment]
I thought this file was somehow generated by the modifications I made in this pull request: `translations\cldr\main--da-DK--currencies`

I made changes to this file and tested my changes.
[main--da-DK--currencies.zip](https://github.com/PrestaShop/PrestaShop/files/2273938/main--da-DK--currencies.zip)

**Comment by MathiasReker** [comment]
I apply a hotfix for now and wait for 1.7.6 :-)

File: localization/dk.xml (hunk 1 above)
Comment (eternoendless) [reply localization/dk.xml:124]: You can open a separate Pull Request for that if you consider it should be changed :)

Comment (MathiasReker) [reply localization/dk.xml:124]: sure

**Comment by Mreker** [comment]
done: https://github.com/PrestaShop/PrestaShop/pull/9436

File: localization/dk.xml (hunk 1 above)
Comment (MathiasReker) [reply localization/dk.xml:124]: done: https://github.com/PrestaShop/PrestaShop/pull/9436
